plugins/processors/port_name/port_name.go

```diff
@@ -97,11 +108,26 @@ func readServices(r io.Reader) sMap {
 
 func (d *PortName) Apply(metrics ...telegraf.Metric) []telegraf.Metric {
 	for _, m := range metrics {
-		portProto, ok := m.GetTag(d.SourceTag)
-		if !ok {
-			// Nonexistent tag
+
+		var portProto string
+		var fromField bool
```

Comment:
fromField isn't used.

Reply:
It is used in the end to determine if the looked up service name is to be added as a tag or as a field.